content/en/contribute/medic/product-development-process/ux-research-repo/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "UX Research Repository"
+linkTitle: "UX Research Repository"
+weight: 4
+description: >
+  How to keep track of product research
+---
+
+The Medic Product team uses [baserow.io](http://baserow.io) as a place to keep product research. Baserow is very similar to [airtable.com](http://airtable.com), [getgrist.com](http://getgrist.com), and Google Sheets.
+
+The _UX Research Repository_ is based off of concepts from WeWork’s [Polaris](https://tsharon.medium.com/democratizing-ux-670b95fbc07f) project. You can see an example of the Polaris (which uses Airtable) [here](https://www.airtable.com/universe/expShuhNMi0Oc0xpb/polaris-ux-nuggets).
+
+The Baserow interface can be seen below. The items in the list on the left are “tables” that we have created. On the right is the “Grid”. It looks just like a spreadsheet and you can have multiple views of the same grid.
+
+![The Baserow interface 👆🏼](images/baserow.png)
+
+To gain access to the UX Research Repo, please reach out in the #product-owner-chatter Slack channel. The access to the repository is currently restricted to the Medic team. 
+
+## Quick Links
+
+- [See if an **Organization** exists](https://baserow.io/database/28935/table/65708)
+    - [Add a new **Organization**](https://baserow.io/form/1L-6KrFryozsBW64-5hnPG6b5zh5HsBjPwPt6t0bNR4)
+- [See if an **Interview Subject** already exists](https://baserow.io/database/28935/table/65706) (*hint: 🔍 in the upper right of Baserow*)
+    - [Add a new **Interview Subject**](https://baserow.io/form/leQCgnXauYXdpJUxtMFBY6CHh_rT_JupsuZC67xvrBQ)
+- [Add a new **Sample** to an existing Interview Subject](https://baserow.io/form/fbePtwPQt5GRxaFWcV5yL31OQJQaLW5HifFRFhy9Kwc)
+- [Add a new **Nugget** to an existing Sample](https://baserow.io/form/6ABmeIk87R8vTdYVfoGsV3hm2DPQkPkv1aNK1WR_dUg)
```

Comment:
These are all accessible to the public with needing a Baserow login.  When it was gated in Notion, this was fine because they were public, but un-guessable URLs. With direct links, the public can either think they're invited to add thoughtful submissions (best case) or spam us with their commercial offerings (worst case).

@michaelkohn  - thoughts on this? you've done a lot more digging into baserow's features.

If we can't think of any options, I think we should remove all the "Add a new..." links and instead post a link to [Notion's quick links page](https://www.notion.so/medicmobile/UX-Research-Repository-78c4153f21f04f97bea2025d99c09836) (or a simplified version of same)

Reply:
Normally I would be fine with exposing them publicly and seeing if it becomes a problem, but some of the forms have dropdowns with private data in them. It is possible to restrict access to the forms with a password, but I think that'll probably ask for it every time, which will be annoying for the people who will be using this a lot. Let me experiment with that later today.  The other option is to replace the links to the forms with links to the tables, which  requires authentication and will keep you logged in throughout your session.

Reply:
Interestingly... I was able to password protect each form and it only every asked for the password once (for each form) as long as i didn't change the password, which is "convenient" for our use case. That said, rather than complicate it, i'd suggest removing the links and letting folks access in a way that is meaningful for them. 



![Screenshot 2023-01-19 at 1 13 20 PM](https://user-images.githubusercontent.com/10096985/213526829-9a6da467-06fd-48aa-8552-6bd9b5de7260.png)
way.

Reply:
Thanks both! I agree with removing the links, I don't think they bring much value to the documentation. The key concepts such as `organisation` and `nuggets` are explained in the `/key-concepts` page, and I feel that it is sufficient to give context to the readers. As a consequence, I removed the links.